### DISCUSSION TITLE
SSL Certificate out of date

### DISCUSSION THREAD

**Discussion by rexsum420** [mitmproxy/mitmproxy#6698] [General]
I just downloaded mitm and when i tried to have my browser trust the ssl certificate, my certificate is showing out of date as of Feb 27, 2024. Is there a new certificate i can download or is there a way to update this one?

**Comment by Prinzhorn** [comment]
What version? Do you not get an error from mitmproxy when launching? Did you ever use mitmproxy before? There could be an old cert still be in `~/.mitmproxy/` , but if it's expired, you should get an error when launching mitmproxy.

**Reply by rexsum420** [reply]
No I haven't used mitmproxy before but I have version 10.2.2 and it loads up and runs with no errors but yea the `mitmproxy-ca-cert.cer` file is showing expired Feb 27, 2024 and I tried running `sudo update-certificates` but that didn't do anything with the mitmproxy certificate. The only error I get is when I have the cert trusted in my browser and the proxy set, then the browser tells me the certificate is expired

**Reply by Prinzhorn** [reply]
> mitmproxy-ca-cert.cer file is showing expired Feb 27, 2024 

And that file was created today? And your system clock isn't/wasn't messed up?

If you delete the files from `~/.mitmproxy/` and start mitmproxy again, what is the expiration date of the new cert?

**Reply by rexsum420** [reply]
Oh wow that pretty easy, so the new certificate says it expires 02/27/2034 so I'm going to try it later and it should work. Thank you.
